app/domain/authentication/security/extract_reques_remote_addr_info.rb

```diff
@@ -0,0 +1,51 @@
+require 'command_class'
+
+module Authentication
+  module Security
+    ExtractRequestRemoteAddrInfo = CommandClass.new(
+      dependencies: {
+        # InputValidation
```

Comment:
what's this? when will this happen?

Reply:
also - which input are we validating?

Reply:
It was planned that Moti will fo that, now I don't know. It should validate the IPs.